plasma/plasma-mvp/specifications/morevp.md

```diff
@@ -0,0 +1,655 @@
+
+
+#### Priority
+
+The above game correctly selects the inputs or outputs that are eligible to exit.
+However, we still need to enforce an ordering on exits to ensure that all outputs created by valid transactions will be paid out before any output created by an invalid transaction.
+We do this by ordering every exit by the position of the *youngest input* to the transaction referenced in each exit, regardless of whether an input or an output is being exited.
+
+
+## Exit Protocol
+
+### Motivation
+
+The MoreVP exit protocol described above guarantees that correctly behaving users will always be able to withdraw any funds they hold on the Plasma chain.
+However, we avoided describing how the users actually prove the statements they’re required to prove.
+This section presents a specification for an implementation of the exit protocol.
+The implementation is designed to be deployed to Ethereum and, as a result, some details of this specification take into account limitations of the EVM.
+
+Additionally, the MoreVP exit protocol is not necessary in all cases.
+Previous work shows that we can use the Plasma MVP exit protocol without confirmation signatures for any transaction included before an invalid (or, in the case of withheld blocks, potentially invalid) transaction.
+We therefore only need to make use of the MoreVP protocol for the set transactions that are in-flight when a Plasma chain becomes byzantine.
+
+The MoreVP protocol guarantees that if transaction is exitable then either the unspent inputs or unspent outputs can be withdrawn.
+Whether the inputs or outputs can be withdrawn depends on if the transaction is canonical.
+However, in the particular situation in which MoreVP exits are required, users may not be aware that an in-flight transaction is actually non-canonical.
+This can occur if the owner of an input to an in-flight transaction is malicious and has signed a second transaction spending the same input.
+
+To account for this problem, we allow exits to be ambiguous about canonicity.
+Users can start MoreVP exits with the *assumption* that the referenced transaction is canonical.
+Other owners of inputs or outputs to the transaction can then “piggyback" the exit.
+We add cryptoeconomic mechanisms that determine whether the transaction is canonical and which of the inputs or outputs are unspent.
+The end result is that we can correctly determine which inputs or outputs should be paid out.
+
+
+### MoreVP Exit Protocol Specification
+
+#### Timeline
+
+The MoreVP exit protocol requires the use of a “challenge-response” mechanism, whereby users can submit a challenge but are subject to a response that invalidates the challenge.
+To give users enough time to respond to a challenge, the exit process is split into two “periods.” When challenges are subject to a response, we require that the challenges be submitted before the end of the first exit period and that responses be submitted before the end of the second.
```

Comment:
would add: 
1) why a challenge is needed in the first place (e.g., to ensure that the rootchain is getting accurate information re: our talk earlier this week)
2)  why MFP is 7 days (re: maximum time required for all txns to exit if chain is byzantine)

this is something many don't understand (e.g., Rick Dudley) and will directly impact how readers understand the UX of plasma, best to have it laid out that this is a valuable mechanism timeline